pipeline-model-definition/src/test/resources/whenEnvFalse.groovy

```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License
  *
- * Copyright (c) 2016, CloudBees, Inc.
+ * Copyright (c) 2017, CloudBees, Inc.
```

Comment:
Well... the file only changed name, so the year shouldn't really be updated. But that's just a really _anal nit_ :)